apache-apisix/manifest.json

```
@@ -0,0 +1,45 @@
+{
+  "display_name": "Apache APISIX",
+  "public_title": "Datadog-APISIX Integration",
+  "integration_id": "apache-apisix",
+  "is_public": true,
+  "maintainer": "dev@apisix.apache.org",
+  "manifest_version": "1.0.0",
+  "name": "apache-apisix",
+  "metric_prefix": "apisix.",
+  "metric_to_check": [
+    "apisix.request.counter",
+    "apisix.request.latency",
+    "apisix.upstream.latency",
+    "apisix.apisix.latency",
+    "apisix.ingress.size",
+    "apisix.egress.size"
+  ],
+  "creates_events": false,
+  "short_description": "Datadog-APISIX Integration",
+  "description": "Datadog-APISIX integration using DogStatsD",
+  "guid": "fb69375e-5742-42e6-81cf-8fc19de610ed",
+  "support": "contrib",
+  "supported_os": [
+    "windows",
+    "mac_os",
+    "linux"
+  ],
+  "categories": [
+    "monitoring",
+    "web",
+    "cloud",
+    "log collection",
+    "autodiscovery",
```

```suggestion
    "cloud",
    "autodiscovery",
```
https://github.com/DataDog/integrations-extras/pull/1100/checks#step:13:12

---

```suggestion
    "cloud",
    "log collection",
```
Autodiscovery is usually when the agent creates instances of an integration itself, which is not the case here since APISIX sends metrics on its own